/jds/bin/diff -uprN findutils-4.2.31.old/find/find.1 findutils-4.2.31/find/find.1
--- findutils-4.2.31.old/find/find.1	2009-02-27 17:41:47.128604000 +0000
+++ findutils-4.2.31/find/find.1	2009-02-27 17:44:35.423639000 +0000
@@ -1,3 +1,4 @@
+'\" te
 .TH FIND 1 \" -*- nroff -*-
 .SH NAME
 find \- search for files in a directory hierarchy
@@ -1201,6 +1202,20 @@ description, but if the return value is 
 on the correctness of the results of 
 .BR find .
 
+.SH "ATTRIBUTES"
+.PP
+See \fBattributes\fR(5)
+for descriptions of the following attributes:
+.sp
+.TS
+tab(^G) allbox;
+cw(2.750000i)| cw(2.750000i)
+lw(2.750000i)| lw(2.750000i).
+ATTRIBUTE TYPE^ATTRIBUTE VALUE
+Availability^SUNWgnu-findutils
+Interface stability^Uncommitted
+.TE
+.sp
 .SH "SEE ALSO"
 \fBlocate\fP(1), \fBlocatedb\fP(5), \fBupdatedb\fP(1), \fBxargs\fP(1),
 \fBchmod\fP(1), \fBfnmatch\fP(3), \fBregex\fP(7), \fBstat\fP(2),
@@ -1259,3 +1274,4 @@ the findutils package in general can be 
 .I bug\-findutils
 mailing list.  To join the list, send email to 
 .IR bug\-findutils\-request@gnu.org .
+...\" LSARC 2008/531 findutils for OpenSolaris
/jds/bin/diff -uprN findutils-4.2.31.old/xargs/xargs.1 findutils-4.2.31/xargs/xargs.1
--- findutils-4.2.31.old/xargs/xargs.1	2009-02-27 17:41:48.032285000 +0000
+++ findutils-4.2.31/xargs/xargs.1	2009-02-27 17:45:01.473882000 +0000
@@ -1,3 +1,4 @@
+'\" te
 .TH XARGS 1 \" -*- nroff -*-
 .SH NAME
 xargs \- build and execute command lines from standard input
@@ -225,6 +226,20 @@ The \-l and \-i options appear in the 19
 standard, but do not appear in the 2004 version of the standard.
 Therefore you should use \-L and \-I instead, respectively.
 
+.SH "ATTRIBUTES"
+.PP
+See \fBattributes\fR(5)
+for descriptions of the following attributes:
+.sp
+.TS
+tab(^G) allbox;
+cw(2.750000i)| cw(2.750000i)
+lw(2.750000i)| lw(2.750000i).
+ATTRIBUTE TYPE^ATTRIBUTE VALUE
+Availability^SUNWgnu-findutils
+Interface stability^Uncommitted
+.TE
+.sp
 .SH "SEE ALSO"
 \fBfind\fP(1), \fBlocate\fP(1), \fBlocatedb\fP(5), \fBupdatedb\fP(1),
 \fBFinding Files\fP (on-line in Info, or printed)
@@ -284,3 +299,4 @@ the findutils package in general can be 
 .I bug\-findutils
 mailing list.  To join the list, send email to 
 .IR bug\-findutils\-request@gnu.org .
+...\" LSARC 2008/531 findutils for OpenSolaris
